share/locale/languages.json

```diff
@@ -59,7 +59,8 @@
     "sl": "Slovenščina",
     "sr": "Srpski",
     "sr_RS" : "Српски",
-    "sv": "Svenska",
+    "sv": "Svenska (modern)",
+    "sv_SE": "Svenska (traditionell)",
```

Comment:
Shouldn't the terms 'modern' and 'traditionell' be Swedish terms? Or are they already

Reply:
They are already. English is 'tradition<b>al</b>'.

Reply:
Ouch, yes